performance/src/gatling/simulations/org/pmiops/databrowser/Pages.scala

```diff
@@ -0,0 +1,23 @@
+package org.pmiops.databrowser
+
+import io.gatling.core.Predef._
+import io.gatling.core.structure.ChainBuilder
+import io.gatling.http.Predef._
+
+object Pages {
+
+  object Home {
+    val home: ChainBuilder = exec(http(HomePage.name)
+      .get("/")
+      .check(status.is(session => 200)))
+      .pause(Configuration.defaultPause)
+  }
+
+  object Search {
+    val search: ChainBuilder = Home.home
```

Comment:
I'm not quite following the dependency between Home and Search here. Is this basically saying "go to Home, then go to search?"

Reply:
It is exactly that. I agree this could be more clear - will do some more thinking along those lines as I do the scenario refactoring.